source/install.txt

```diff
@@ -101,6 +101,11 @@ Procedure
       
             .. include:: /includes/steps/install-shell-linux-generic.rst
 
+.. important:: Windows Users
+
+   ``mongosh`` preferences are stored by default in the directory
```

Comment:
As part of the same ticket (DOCSP-13741), there is also an update to where log files are stored. We may also need to update https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-13662/logs accordingly?